src/components/20-molecules/file-upload/story.js

```diff
@@ -79,6 +79,7 @@ export const FileUpload = () => {
     'The following files are being transferred:'
   );
   const preventFileCompression = boolean('preventFileCompression', false);
+  const invalid = boolean('invalid', false);
```

Comment:
I personally wouldn't put the new attribute into the story because it's read only. The `outputdate` attribute from the datepicker isn't is the story either (you can look it up).

Reply:
Rather make a good demo that shows the value of the `invalid` attribute when the user uploads a file